Directory.Build.props

```diff
@@ -35,8 +35,8 @@
          * Major/Minor match Android stable API level, such as 30.0 for API 30.
          * Bump first digit of the patch version for feature releases (and reset the first two digits to 0)
     -->
-    <AndroidPackVersion>35.0.0</AndroidPackVersion>
-    <AndroidPackVersionSuffix>rc.2</AndroidPackVersionSuffix>
+    <AndroidPackVersion>35.0.1</AndroidPackVersion>
```

Comment:
This change will reset our commit delta portion of the version, and I was wondering if there would be a good way to save this "version reset" for closer to when we branch RTM.

Maybe it makes sense to remove the `main` branch check from https://github.com/dotnet/android/blob/e9ee3d4cc35edd5986629d2d5a5415ff93dbe0d5/build-tools/scripts/XAVersionInfo.targets#L82 at this point? That would cause main to start so to produce packages with the following prelease label instead:

    *.ci.main.$(CommitCount)*

And we could reserve the rtm branding/labeling for release branches